pyensembl/locus.py

```diff
@@ -1,10 +1,10 @@
 def normalize_chromosome(c):
-    if isinstance(c, (int, long)):
+    if isinstance(c, int):
```

Comment:
Yeah, this was another one I wasn't thrilled about. But, an int in Python 2 is implement as a C "long", which on any modern platform is 64 bits (and guaranteed to be at least 32 bits), so it's really pretty unlikely to need a Python long here.